ckanext/ldap/controllers/user.py

```diff
@@ -287,6 +287,8 @@ def _find_ldap_user(login):
 
     filter_str = config[u'ckanext.ldap.search.filter'].format(
         login=ldap.filter.escape_filter_chars(login))
+    filter_str_alt = config[u'ckanext.ldap.search.alt'].format(
+        login=ldap.filter.escape_filter_chars(login))
```

Comment:
`ckanext.ldap.search.alt` is optional so I'm pretty sure this will raise an exception if it's not defined in the config.